### ISSUE TITLE
[1.15.2] IE Crash - Exception in server tick loop - Game crashes when approaching Nether Portal on Nether side / Game crashes while in the End for a varying amount of time

### ISSUE THREAD

**Issue by Kaligraphy22** [BluSunrize/ImmersiveEngineering#4368]
Description of the issue:
IE crashes whenever I get too close to my Nether Portal on the Nether side or if I'm in the End for a few minutes. Let me take a moment to apologize that I don't really understand or know how to read crashlogs effectively. I did read mine (there were several, not just the one I'm uploading) but all I could understand that was that this mod had something to do with it. If it didn't, then I apologize for wasting your time. I swore I saw a similar crash about wires placed near chunk borders for previous IE versions but I do not know if this was the same issue I am currently experiencing. And even then, I thought that issue was fixed but as I said before, I'm not good at reading crash logs.

I just went to defeat the Ender Dragon yesterday and I kept crashing while in the End. Each crash felt like it was maybe 10 minutes (less or more) apart. I was equipped with Terrasteel armor from Botania and the chestplate had my capacitor backpack attached to it. I was using the Heavy Plated Shield, too. The game also crashed when I tried to go through the End portal after the fighter concluded to get back to the Overworld. Two of the crashes also deleted my Rod of Bifrost and Sorjourner's Sash from Botania (I have no idea why). I unattached the capacitor backpack and put away my shield thinking maybe that was the issue somehow but the crashes kept coming.

As for the Nether issue, strangely enough, it seems to only happen when getting close to my portal from the Nether side and not the Overworld side. If it also helps, I have a windmill/waterwheel set up near the Nether portal. They are not in the same chunk. They might be diagonal from each other though.

Update: The End is literally unplayable now. I have to quit the world almost every minute and start it back up to get anything done before the game crashes.

### Crashlog:
https://pastebin.pl/view/02349be5

### Versions & Modlist
**Forge Version: forge-1.15.2-31.2.30-installer**

Actually+Useful+Stonecutter-0.0.0+mc+1.15.2
additional_lights-1.15.2-2.0.2
AdFinders-1.15.2-4.0.5.0-build.0020
animalnet-v1.15.2+Forge+b1
AppleSkin-mc1.15.2-forge-1.0.14
AutoRegLib-1.5-40
BetterCaves-1.15.2-1.0
BetterMineshaftsForge-1.15.2-1.3.1
BiomesOPlenty-1.15.2-10.0.0.366-universal
BonsaiTrees-2.1.2.6
Bookshelf-1.15.2-5.6.40
Botania-r1.15-388
BotanyPots-1.15.2-2.0.27
Bountiful-1.15.2-3.1.2
caelus-FORGE-1.15.2-2.0-beta3
Charm-1.15.2-1.5.9
Charmonium-1.15.2-1.0.1
ChunkNoGoByeBye-1.15.2-1.2.0
comforts-FORGE-1.15.2-2.0.0.4
config-2-3.0
CookingForBlockheads_1.15.2-8.0.3
curioofundying-FORGE-1.15.2-3.0
curios-FORGE-1.15.2-2.0.2.6
curiouselytra-FORGE-1.15.2-2.0
CutAndColored-2.0_MC1.15.2
DarkUtilities-1.15.2-3.1.8
decorative_blocks-1.15.2-1.7
DoggyTalents-1.15.2-1.15.2.3
dungeons-plus-1.15.2-1.0.2
elevatorid-1.15.2-1.7.1
FarmingForBlockheads_1.15.2-6.1.5
FastFurnace-1.15.1-3.0.0
FastWorkbench-1.15.2-3.1.1
foodpouches-1.15.2-1.0.1
ForgeEndertech-1.15.2-6.0.3.2-build.0042
globalxp-1.15.2-v1.4.11
grindr-1.15.2-0.0.1.5
guardvillagers-1.15.2-1.0.8
Hwyla-forge-1.10.8-B72_1.15.2
**ImmersiveEngineering-1.15.2-4.1.0-122**
jei-1.15.2-6.0.3.15
Kottle-1.5.2
lanterncolors-1.15.2-1.0.1
libnonymous-1.15.2-1.1.1.5
light-overlay-4.7.2
mcw-bridges-1.0.4fix-mc1.15.2
mobcatcher-2.2
NaturesCompass-1.15.2-1.8.5
packedup-1.0.16-mc1.15
pamhc2foodcore-1.15.2-1.1.0
Patchouli-1.15.2-1.2-35
Placebo-1.15.2-3.1.0
polymorph-FORGE-1.15.2-0.10
QuarkOddities-1.15.2
Quark-r2.1-245
SaveMyStronghold-1.15.2-1.0.1
SilentLib-1.15.2-4.6.5+58
simpleplanes-1.15.2-3.0.0.2
StorageDrawers-1.15.2-7.0.2
Strange-1.15.2-1.0.0-beta.5.hotfix1
structure-gel-api-1.15.2-1.1.0
TerraForged-1.15.2-0.0.15
tesseract-1.0.11-mc1.15
TinyMobFarm-1.15.2-1.0.8
tombstone-4.5.6
ToolBelt-1.15.2-1.14.1
TorchBandolier-1.15.2-1.4.0+8
trashcans-1.0.1-mc1.15
WaterStrainer-1.15.2-9.0.0
Xaeros_Minimap_20.20.3_Forge_1.15.2
XaerosWorldMap_1.10.3_Forge_1.15.2

**Comment by Kaligraphy22** [comment]
Update: So I took out all the wires in my base. I only had MVs and HVs. I left the connectors. I went to the end and sat there for a while. Still crashed with the same error. But this time, I noticed something to do about the Quark mod but I'm not sure what. I also saw something about the Tesseract mod. I do use that mod to wirelessly transfer power from my disel generator setup to my Arc Furnace and Crusher. Both tesseracts have a HV connector attached to it. One for the disel generator and one for the arc furnace and crusher.

As for Quark, as I said before, I don't know why it was mentioned in the crash log. I will say that there are some connectors attached to a quark block. Just a simple decorative block like polished jasper. I don't know why that would be an issue. I had one feed-through block for HV wires. Breaking this did not stop the crash. I don't know what else to do. And the strangest part of this? This crash doesn't occur in my test world. I keep a test world for, well testing mods before I play with them in my survival world. I went to the End, stayed there for a while. And no crash. I even explored the outer islands and everything was fine. Note: I purposefully put the end portal (in my test world) near an excavator with HV wires to see if it was some weird proximity issue. Nope. Nothing. No crash. Nothing.

**Comment by Kaligraphy22** [comment]
Update: So I removed every connector and tesseract from my survival world. No crashes. I can't really play IE without using connectors. So... I can play in the End again at the cost of not being able to play with IE's machines. I guess I could use Creative capacitors on everything but that defeats the purpose of my power setup. :( I'd really like some help with this issue if anyone can spare the time.

**Comment by BluSunrize** [comment]
Please test if this issue persists with the new release of IE:

https://www.curseforge.com/minecraft/mc-mods/immersive-engineering/files/3099023

**Comment by Kaligraphy22** [comment]
Hello! I'm happy to say that the issue seems to have been fixed with the new release for 1.15.2. I re-added all my connectors and wires then I stayed in the End for a long time and not one crash. I did not test the Nether yet. Unfortunately, I fell asleep at the keyboard (while in the End) and it's 1:39am now... but I imagine that should be fixed, too. Just to be sure, I'll test when I get home after work today.

However, I did run into another strange issue involving the Diesel generators that I believe I'm supposed to make a new request for but I'd still like to bring it to your attention before I forget. The generators seem to stay on and burn fuel with only the connectors attached to them. They run until either I break all the connectors or I turn them off with a redstone signal, burning all that diesel. No wires required. I saw this while I was re-connecting my power network to see if the previous issue had been fixed. I know they didn't do this before. They usually stop unless the power is needed. My HV capacitors were full and all machines were full on internal power and not in use. This had nothing to do with the Tesseract mod as I had not re-connected that particular connecting at the time.

Regardless, I will test the Nether when I get home and let you know. Thank you very much for taking the time to help with my issue.
